v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent [`INSERT`](insert.html), [`UPDATE`](update.html), [`UPSERT`](upsert.html), and [`DELETE`](delete.html) statements within a single [transaction](https://www.cockroachlabs.com/docs/dev/transactions.html). Executing statements in parallel helps reduce aggregate latency and improve performance. 
+
+<div id="toc"></div>
+
+## Why Use Parallel Statement Execution
+
+To understand why parallel statement execution is required, let's consider a sample scenario to demonstrate how SQL statements are executed. 
+
+Consider a scenario where a user's last name, favorite movie, and favorite song are to be updated on a social networking application. Suppose the database has three tables: username, movies, and songs, that need to be updated to update the user information. Also suppose that the user's ID is 1. Then the traditional transaction to update the user's information is as follows:
+
+~~~ sql
+> BEGIN;
+> UPDATE username SET last_name = 'Smith' WHERE id = 1;
+> UPDATE movies SET favorite_movie = 'The Matrix' WHERE id = 1;
+> UPDATE songs SET favorite_song = 'All this time' WHERE id = 1;
+> COMMIT;
+~~~
+
+In this case, the statements are executed sequentially. Sequential execution of SQL statements returns a return value immediately from the server. After the return value is received, the next SQL statement is executed. The following conceptual diagram shows the sequential execution of the statements.
+
+<img src="{{ 'images/Sequential_Statement_Execution.png' | relative_url }}" alt="CockroachDB Parallel Statement Execution Error Mismatch" style="border:1px solid #eee;max-width:100%" />
```

Comment:
I don't think we need the "transaction" bracket in the diagrams. It's just extra clutter and the Begin/Commit statements already make it very clear.

Reply:
Hmm..I agree..let's check with @jseldess though.

Reply:
Yep, removing those brackets works for me, too!